src/parsers/trades/liquid/index.ts

```diff
@@ -0,0 +1,175 @@
+import { getCSVData } from '../../';
+import { EXCHANGES, IImport, IPartialTrade, ITrade } from '../../../types';
+import { createDateAsUTC, createID } from '../../utils';
+
+enum LiquidOrderDirection {
+    PAY = 'PAY',
+    RECEIVE = 'RECEIVE',
+}
+
+interface ILiquid {
+    exchange: string;
+    currency_type: string;
+    direction: string;
+    transaction_id: string;
+    transaction_type: string;
+    gross_amount: string;
+    currency: string;
+    execution_id: string;
+    generated_for_type: string;
+    generated_for_id: string;
+    transaction_hash: string;
+    from_address: string;
+    to_address: string;
+    state: string;
+    created_at_jst: string;
+    updated_at_jst: string;
+    created_at_utc: string;
+    updated_at: string;
+    notes: string;
+    used_id: string;
+    account_id: string;
+}
+
+function groupByExecutionID(group: any, line: any) {
+    group[line.execution_id] = group[line.execution_id] ?? [];
```

Comment:
I think we should be able to define both of these types.

Line I would assume is a Iliquid type? And group would be something like this I think:

```
interface group {
 [key: string]: Iliquid[];
}
```

Reply:
True. Done.